### PR TITLE
OQ-10-Create-a-Form

### PR DESCRIPTION
**Pull request by golnazsaraji**
it's actually about showing the list of service types to the customer

### CODE REVIEW

File: client/src/Components/Customer/Service/Service.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: 'Gloria Hallelujah';
```
Comment (lbordino) [inline client/src/Components/Customer/Service/Service.scss:2]: can u change the font to something more "classic"

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '/vite.svg';
+// import reactLogo from './assets/react.svg';
```
Comment (lbordino) [inline client/src/App.tsx:2]: delete

File: client/src/Components/Customer/Service/Services.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import { Service } from '../../../Models/service'
+import { Button, Card } from 'react-bootstrap';
+
+function GetServices() {
+
+    const [serviceList, setServiceList] = useState<Service[]>([]);
+    const [error, setError] = useState<string>("");
+    const [ticket, setTicket] = useState<string>("");
+    const [ticketIsRequested, setticketIsRequested] = useState<boolean>(false);
+    const [serviceId, setServiceId] = useState<number>(0);
```
Comment (lbordino) [inline client/src/Components/Customer/Service/Services.tsx:14]: are we sure that there is a 0 service? Whenever the page refresh it sends 2 calls to get service 0

File: server/src/errors/servicesError.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+const Service_ALREADY_EXISTS = "The service type is already exists";
+
+class ServiceAlreadyExistsError extends Error {
+    customMessage: String;
+    customCode: Number;
+
+    constructor() {
+        super()
+        this.customMessage = Service_ALREADY_EXISTS
+        this.customCode = 401;
```
Comment (lbordino) [inline server/src/errors/servicesError.ts:11]: this is a 409 conflict

File: server/src/routes.ts [hunk 5]
```diff
@@ -1,14 +1,16 @@
...
 import ticketRoutes from './routers/ticketRoutes';
 import { app } from '../index';
+import serviceRoutes from "./routers/serviceRoutes";
 
 
 function initRoutes(app: Application) {
-    app.get("/queue/number", async (req, res) => {
-        res.status(200).json({ ok: "ok" });
-    });
+    // app.get("/queue/number", async (req, res) => {
```
Comment (lbordino) [inline server/src/routes.ts:8]: delete

**Review by lbordino — CHANGES_REQUESTED** [review]
Remove all the unused code, font and images. I've attached some comments to refactor the code

File: client/src/App.tsx (hunk 2 above)
Comment (golnazsaraji) [reply client/src/App.tsx:2]: done

File: server/src/routes.ts (hunk 5 above)
Comment (golnazsaraji) [reply server/src/routes.ts:8]: done

**Review by lbordino — APPROVED** [review]

**Comment by golnazsaraji** [comment]
create the service list for customer

**Pull request merged**